### DISCUSSION TITLE
Chives and Plots

### DISCUSSION THREAD

**Discussion by Yamanipanuchi** [guydavis/machinaris#357] [Q&A] [answered]
I have Chives setup like the other coins, It doesnt see my plots. Is it possible that Chives does like NFT plots and I missed that info somewhere?

**Comment by guydavis** [comment, marked as answer]
Hi, Chives plots are different than regular Chia plots. As such, the default Machinaris configs put them into a child folder on your host OS path named "chives/". 

See the wiki on Chives page. on my phone so terse

**Reply by Yamanipanuchi** [reply]
Ahh, yes I see. So for me its not worth running chives right now. Thanks!

**Reply by guydavis** [reply]
Yes.  Totally up to you as each fork available to Machinaris is optional.  I am farming just a handful of small Chives plots (k31) to fill up extra space on some 8 TB drives where I can't fit anymore full-sized Chia plots (k32).   Here's what it looks like:

![image](https://user-images.githubusercontent.com/12936/138527637-d09316b5-393c-4bab-8c24-b456eb962bfb.png)

![image](https://user-images.githubusercontent.com/12936/138527684-558c84cf-bb35-45fa-84ca-114670a5b3e2.png)

**Reply by jimuyouyou** [reply]
Clever idea! I will do so to utilize my disk space. Thanks so much!
